website/versions/source-snyk.json

```diff
@@ -0,0 +1 @@
+{ "latest": "plugins-source-snyk-v1.0.0" }
```

Comment:
I don't think we need to add this file, it will be generated by our CI

Reply:
removed